package.json

```diff
@@ -63,6 +63,19 @@
     "@react-keycloak/web": "^3.4.0",
     "@reduxjs/toolkit": "^1.6.2",
     "@testing-library/jest-dom": "^5.15.0",
+    "@types/file-saver": "^2.0.2",
+    "@types/loadable__component": "^5.13.4",
+    "@types/react-router-dom": "^5.3.2",
+    "@types/d3": "^7.1.0",
+    "@types/lodash": "^4.14.178",
+    "@types/node": "^16.11.7",
+    "@types/react": "^17.0.34",
+    "@types/react-dom": "^17.0.11",
+    "@types/react-highlight-words": "^0.16.4",
+    "@types/react-router": "^5.1.17",
+    "@types/redux-logger": "^3.0.9",
+    "@testing-library/react": "^12.1.2",
```

Comment:
Unfortunately, I had no choice to this. Suggestion: two ts config files: 1 for prod, 1 for dev and excluding test files in production.